package.json

```diff
@@ -91,6 +92,7 @@
     "express-static-gzip": "^2.0.5",
     "gnu-terry-pratchett": "0.0.2",
     "helmet": "^3.21.1",
+    "isarray": "^2.0.5",
```

Comment:
not too sure how it works - the package-lock still has multiple versions of `isarray`

isn't it package-lock that we need to update in order to pin transitory dependencies?

Reply:
This is installed to guarantee theres a version to alias too